### PR TITLE
Hyperfine: Script re-org

### PR DESCRIPTION
**Pull request by gunnarmorling**
This splits up the existing launch scripts to adhere to the structure established with https://github.com/gunnarmorling/1brc/pull/182. Only to be merged once that one is in!

@hundredwatt, @AlexanderYastrebov, review welcome. This was a bit of busy work, I hope I didn't mess up anything.

### CODE REVIEW

File: calculate_average_iziamos.sh [hunk 1]
```diff
@@ -17,4 +17,4 @@
 
 
 JAVA_OPTS="--enable-preview"
-time java $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_iziamos
+tjava $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_iziamos
```
Comment (AlexanderYastrebov) [inline calculate_average_iziamos.sh:20]: tjava

**Comment by AlexanderYastrebov** [comment]
Since you change all of them, maybe it would be an improvement to move them into `./benchmark` or `./bench` before its too late (I think it would be easy to adjust eval scripts).
But this could also be done in subsequent PRs in bulk after this prepare/calculate split.

**Comment by AlexanderYastrebov** [comment]
I think you'd also like to update README.md

**Comment by gunnarmorling** [comment]
Fixed the typo and updated the README. I'm leaving that re-org for a separate PR, to keep things manageable. Thanks a lot for reviewing, @AlexanderYastrebov.

**Comment by gunnarmorling** [comment]
Tested with a few entries, all looking good. Merging.

**Pull request merged**